src/js/components/AppDebugInfoComponent.jsx

```diff
@@ -63,6 +63,9 @@ var AppDebugInfoComponent = React.createClass({
     }
 
     const timestamp = lastTaskFailure.timestamp;
+    const timeStampText = new Date(timestamp) > new Date()
+      ? "Just Now"
```

Comment:
Maybe "Just now" ? Because this is just usual text. On a button or such, I would prefer "Just Now" too.